### PR TITLE
Countdown Widget and Twitter Widgets

### PR DESCRIPTION
**Pull request by Abdulla-Shahid**
## Description

The following PR involves adding more widgets to the Home Page + fixing its layout. This adds both twitter widgets (DukeMMB and KvilleNation) to the home page and also adds a dynamic countdown timer for the Duke v.s UNC game. It also switches the overall layout of the homepage from a container to a grid for better organization/widget layout. 

## Motivation and Context

Need a nice UI + helpful widgets on the homepage to increase the likelihood someone would use this application. 
## How Has This Been Tested?

Relatively static content, so testing hasn't been substantial yet (Will be tested more once we as a group start the testing phase). Testing was done through the console to make sure there weren't any errors or resizing issues. The accuracy of the countdown widget was compared to an online timer to make sure the date being shown was correct. 
1. Yarn dev:start
2. Login/Sign Up 
3. Go to Home Page on Top

## Screenshots (if appropriate):

screen shot 2019-01-04 at 2 58 14 am

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Vinitowinit** [comment]
I am unable to properly review your PR. Before I ran yarn dev:start, I git pulled, checkout to your branch, and yarn installed. This is the error I receive on the server side. 
screen shot 2019-01-04 at 8 55 01 am

**Comment by amanmibra** [comment]
@Vinitowinit try to solve the problem before posting a review. This is on you once again. Read the error messages it clearly says `GemNotFound`, probably because you have not installed it. Try to google your problem and debug yourself first or privately message Abdullah. This error is not worth a review post.

Just to save you time, to solve your problem run:
`docker-compose build`. 

And in case, you need to migrate new data. `yarn dev:db:migrate`.

File: app/javascript/src/components/home/HomeBody/index.jsx [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
+import CDWidget from './CDWidget'; 
 
-  render () {
-    const { user } = this.props.userData;
-    const { visible } = this.state;
+const HomeGrid = () => (
+  <Grid columns='equal'>
+    <Grid.Row>
+      <Grid.Column>
+      <KvilleWeather/>
```
Comment (amanmibra) [inline app/javascript/src/components/home/HomeBody/index.jsx:13]: For readability, please tab/space this line.

**Review by amanmibra — APPROVED** [review]
Looks good to me! Nice work, only change I ask is a syntactical one.

**Comment by amanmibra** [comment]
I went ahead and git pull staging for you!

**Pull request merged**